patterns/vpc-lattice/cross-cluster-pod-communication/cluster/eks.tf

```diff
@@ -0,0 +1,270 @@
+        value = "lattice-gateway" # Specify this parameter to create a default VPC lattice service
+      },
+      # {
+      #   name  = "latticeEndpoint"
+      #   value = "https://vpc-lattice.${local.region}.amazonaws.com"
+      # }
+    ]
+    wait = true
+  }
+  enable_external_dns            = true
+  external_dns_route53_zone_arns = try([data.terraform_remote_state.environment.outputs.route53_private_zone_arn], [])
+  external_dns = {
+    create_role = true,
+    set = [
+      {
+        name  = "domainFilters[0]"
+        value = local.domain
+      },
+      {
+        name  = "policy"
+        value = "sync"
+      },
+      {
+        name  = "sources[0]"
+        value = "crd"
+      },
+      {
+        name  = "sources[1]"
+        value = "ingress"
+      },
+      {
+        name  = "txtPrefix"
+        value = module.eks.cluster_name
+      },
+      {
+        name  = "extraArgs[0]"
+        value = "--crd-source-apiversion=externaldns.k8s.io/v1alpha1"
+      },
+      {
+        name  = "extraArgs[1]"
+        value = "--crd-source-kind=DNSEndpoint"
+      },
+      {
+        name  = "crdSourceApiversion"
+        value = "externaldns.k8s.io/v1alpha1"
+      },
+      {
+        name  = "crdSourceKind"
+        value = "DNSEndpoint"
+      }
+    ]
+  }
+
+  tags = local.tags
+}
+
+module "eks_blueprints_addon_kyverno" {
+  source  = "aws-ia/eks-blueprints-addon/aws"
+  version = "~> 1.1" #ensure to update this to the latest/desired version
+
+  chart            = "kyverno"
+  chart_version    = "3.0.5"
+  repository       = "https://kyverno.github.io/kyverno"
+  description      = "Kyverno"
+  namespace        = "kyverno"
+  create_namespace = true
+}
+
+data "aws_iam_policy_document" "gateway_api_controller" {
+  statement {
+    sid       = ""
+    effect    = "Allow"
+    resources = ["*"] # For testing purposes only (highly recommended limit access to specific resources for production usage)
+
+    actions = [
+      "vpc-lattice:*",
+      "iam:CreateServiceLinkedRole",
+      "ec2:DescribeVpcs",
+      "ec2:DescribeSubnets",
+      "ec2:DescribeTags",
+      "ec2:DescribeSecurityGroups",
+      "logs:CreateLogDelivery",
+      "logs:GetLogDelivery",
+      "logs:UpdateLogDelivery",
+      "logs:DeleteLogDelivery",
+      "logs:ListLogDeliveries",
+      "tag:GetResources",
+    ]
+  }
+}
+
+################################################################################
+# Platform applications
+################################################################################
+
+resource "helm_release" "platform_application" {
+  name             = "platform-${terraform.workspace}"
+  chart            = "./charts/platform"
+  create_namespace = true
+  namespace        = "lattice-gateway"
+  force_update     = true
+
+  #replace = true # This will force a re-deployment
+
+  values = [
+    <<EOF
+    awsAccountID: "${data.aws_caller_identity.current.account_id}"
+    region: "${local.region}"
```

Comment:
```suggestion
```

Reply:
why you suggest to remove region ? this is used in the helm chart to configure vpc lattice envoy container